src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -160,6 +161,7 @@ public DoneCommand getDoneInstrumentCommand(String[] commandComponents, ArrayLis
         doneCommand.setIndex(instrumentNumber);
         return doneCommand;
     }
+    //@@author

```

Comment:
Same here and for the rest, so long got an author tag below there isn't a need to put @@author here